editor/filesystem_dock.cpp

```diff
@@ -1160,9 +1164,7 @@ void FileSystemDock::_update_file_list(bool p_keep_selection) {
 void FileSystemDock::_select_file(const String &p_path, bool p_select_in_favorites) {
 	String fpath = p_path;
 	if (fpath.ends_with("/")) {
-		if (fpath != "res://") {
```

Comment:
Not sure what this change is for?

Reply:
We don't need to remove the slash at the end of the path, otherwise we won't be able to check for the directory. Wherever the method `_select_file()` is called, there is still a slash deletion, if there is one.
The condition
```
if (fpath.ends_with("/")) {
```
is left here to avoid an attempt to load the resource below
```
} else {
        EditorNode::get_singleton()->load_resource(fpath);
    }
```
This is not critical, but an error message appears.
As a result, there is no need to check the condition in the line that you have highlighted.